lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -109,7 +109,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         }
 
         $version = date('YmdHis');
-        $path = $this->generateMigration($configuration, $input, $version, $up, $down);
+        $path = $this->generateMigration($configuration, $input, $version, $up, $down, true);
```

Comment:
@texdc generating a migration should rather be extracted out of the command themselves to allow cleaner logic about configuring the generated migration. But this is best done separately.
for now, this is acceptable (we have worse things in the code of the library btw, and this is an internal function)